docs/features/techdocs/creating-and-publishing.md

```diff
@@ -30,11 +30,6 @@ the documentation template.
 Create an entity from the documentation template and you will get the needed
 setup for free.
 
-!!! warning Currently the Backstage Software Templates are limited to create
-repositories inside GitHub organizations. You also need to generate an personal
```

Comment:
do you want the rest of the warning maintained?

Reply:
@timja I think its fine to delete the entire warning for now! Thanks !